en/02_Developer_Guides/17_CLI/02_Hybrid_Commands.md

```diff
@@ -0,0 +1,172 @@
+---
```

Comment:
Rename file to 02_Poly_Commands.md

Reply:
Renamed to `02_PolyCommand.md` which is what all the links to this page have.